packages/redux-routine/src/index.js

```diff
@@ -12,42 +12,19 @@ import castError from './cast-error';
  * value of the yield assignment. If the control handler returns undefined, the
  * execution is not continued.
  *
- * @param {Object} controls Object of control handlers.
+ * @param {Object}    controls Object of control handlers.
```

Comment:
Minor: We don't want this change. There's no intent to align `@param` descriptions with `@return`.